tests/dags/common/test_sql_helpers.py

```diff
@@ -0,0 +1,128 @@
+import logging
+from datetime import datetime, timedelta
+
+import pytest
+from common.constants import DAG_DEFAULT_ARGS, POSTGRES_CONN_ID
+from common.sql_helpers import PGExecuteQueryOperator, PostgresHook
+from psycopg2.errors import QueryCanceled
+
+from tests.dags.common.test_resources.dags.test_timeout_pg import (
+    TEST_SQL,
+    create_pg_timeout_tester_dag,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def happy_dag():
+    return create_pg_timeout_tester_dag()
+
+
+DEFAULT_TIMEOUT = timedelta(hours=1).total_seconds()
+default_timeout_msg = (
+    f"DAG_DEFAULT_ARGS sets the default task execution timeout to "
+    f"{DAG_DEFAULT_ARGS['execution_timeout']}. Testing for {DEFAULT_TIMEOUT} seconds."
+)
+
+
+# PostgresHook works with or without explicit connection id, and with override conn id
+# Testing mainly for documentation purposes
+# Not using pg.run("select 1;") to test the actual connection, because we don't spin up
+# the databases for testing.
+def test_PostgresHook_init_defaults():
+    pg = PostgresHook()
+    assert pg.postgres_conn_id == POSTGRES_CONN_ID
+    assert pg.default_statement_timeout == DEFAULT_TIMEOUT, default_timeout_msg
+
+
+def test_PostgresHook_conn_id_only_defaults():
+    pg = PostgresHook("gibberish_connection")
+    assert pg.postgres_conn_id == "gibberish_connection"
+    assert pg.default_statement_timeout == DEFAULT_TIMEOUT, default_timeout_msg
+
+
+@pytest.mark.parametrize(
+    "conn_id, timeout, expected_results",
+    [
+        pytest.param(None, None, (POSTGRES_CONN_ID, DEFAULT_TIMEOUT), id="both_None"),
+        pytest.param(
+            "xyzqwerty", None, ("xyzqwerty", DEFAULT_TIMEOUT), id="override_connection"
+        ),
+        pytest.param(None, 60, (POSTGRES_CONN_ID, 60), id="override_timeout"),
+        pytest.param("xyzqwerty", 60, ("xyzqwerty", 60), id="override_both"),
+    ],
+)
+def test_PostgresHook_init_connection(conn_id, timeout, expected_results):
+    (expected_conn_id, expected_timeout) = expected_results
+    pg = PostgresHook(conn_id, timeout)
+    actual_conn_id = pg.postgres_conn_id
+    actual_timeout = pg.default_statement_timeout
+    assert actual_conn_id == expected_conn_id
+    assert actual_timeout == expected_timeout
```

Comment:
This is definitely a stylistic nit, but this could be:
```suggestion
    assert pg.postgres_conn_id == expected_conn_id
    assert pg.default_statement_timeout == expected_timeout
```